workbench/nitro-v2/server/api/trigger.post.ts

```diff
@@ -1,7 +1,7 @@
 import { defineEventHandler, getRequestURL, readRawBody } from 'h3';
 import { start } from 'workflow/api';
 import { hydrateWorkflowArguments } from 'workflow/internal/serialization';
-import { allWorkflows } from '../_workflows.js';
+import { allWorkflows } from '../../_workflows.js';
```

Comment:
This was previously causing `turbo build` to fail for me when run at the root, unsure if relevant